.github/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 <h3>New features since last release</h3>
 
+* `DeviceSpec.layout` and `DeviceSpec.gate_parameters` can now return `None`, which can happen when
+  e.g., a remote simulator device is used.
```

Comment:
Up to you, but I usually find that using more sentences helps with clarity.
```suggestion
* `DeviceSpec.layout` and `DeviceSpec.gate_parameters` may now return `None`. This can happen
  when a remote simulator device is used.
```